README.markdown

```diff
@@ -365,7 +370,12 @@ There are a few known issues on Windows platforms:
 
 * MSI installation failures do not produce any error. If the install fails, puppet_agent continues to be applied to the agent. If this happens, you'll need to examine the MSI log file to determine the failure's cause. You can find the location of the log file in the debug output from either a puppet apply or an agent run; the log file name follows the pattern `puppet-<timestamp>-installer.log`.
 
-* Upgrading on most \*NIX platforms (Linux, AIX, Solaris 11) will end the run after the puppet-agent upgrade finishes. This is to avoid unexpected behavior if already loaded Ruby code happens to interact with newer code that came with the upgrade, or viceversa. If run as a daemon, Puppet will automatically start a new agent run after the upgrade finishes.
+* If the upgrade is from Puppet 5 prior to 5.5.17 or Puppet 6 prior to 6.8.0 to newer version and `msi_move_locked_files` is set to `true`, Puppet can get into a state where `puppet --versions` reports the older version (5.5.16) while the package reported by windows is the new version(5.5.17). To recoved from this case `ADDLOCAL=ALL` must be added to install_options
+``` puppet
+  install_options => ['REINSTALLMODE="maus"', 'ADDLOCAL=ALL']
+```
+
+Upgrading on most \*NIX platforms (Linux, AIX, Solaris 11) will end the run after the puppet-agent upgrade finishes. This is to avoid unexpected behavior if already loaded Ruby code happens to interact with newer code that came with the upgrade, or viceversa. If run as a daemon, Puppet will automatically start a new agent run after the upgrade finishes.
```

Comment:
Can you add a * at the beginning of the line so it's formatted as before

Reply:
Not sure abut this one, as it will make this \*NIX related paragraph appear as part of a Windows enumeration: 
![image](https://user-images.githubusercontent.com/51971875/96869465-2be52280-1478-11eb-8f7d-6bd4e74919ed.png)

Reply:
Oops, missed that. Can we update the section so for each bullet we specify the affected OS, and remove the `There are a few known issues...` line? Historically it looks like the Known issues section was added specifically for Windows, but then I updated it with the *NIX issue, and now it just looks weird.